sdk/sdk-view/src/test/java/mil/emp3/api/CameraLookAtTest.java

```diff
@@ -61,6 +78,20 @@ public void cameraTests() {
         validTiltCameraTest(global.CAMERA_TILT_TO_SKY);
         validHeadingCameraTest(global.HEADING_MINIMUM);
         validHeadingCameraTest(global.HEADING_MAXIMUM);
+        userContext = "Camera test AFTER";
+        mapInstance.applyCameraChange(camera, false, userContext);
+        coreManager.processCameraSettingChange(camera, true, userContext);
+        try {
+            Thread.sleep(100);
+        } catch (InterruptedException e) {
+
```

Comment:
empty catch block.  Should either allow test to fail or comment the block.  I also have an overall question about using Thread.sleep, and using a better alternative

Reply:
Did you ever get a chance to see if you can instead wait for an event to fire?

Reply:
The event does fire, but we have to wait here for the onEvent call to add the user context to the queue after the event fires.  Otherwise we will need to do the assert in an async task with a wait.  That's not worth it because we can do it with a simple sleep.

Reply:
I don't think you can make the junit tests async (correct me if I am wrong),  I believe you will have to do a while loop on a flag until the event fires (which would change the flag then you can perform the assert). The reason I would urge to do this instead of thread.sleep is that it is technically possible for this test to fail by simply checking the assertion too soon.